src/components/MapView/DateSelector/index.tsx

```diff
@@ -172,6 +356,87 @@ const styles = (theme: Theme) =>
     divider: {
       backgroundColor: theme.palette.grey[500],
     },
+
+    dateContainer: {
+      position: 'relative',
+      height: '36px',
+      flexGrow: 1,
+      cursor: 'e-resize',
+      overflow: 'hidden',
+    },
+
+    dateLabelContainer: {
+      position: 'absolute',
+      flexWrap: 'nowrap',
+    },
+
+    timeline: {
+      position: 'relative',
+      top: '5px',
+    },
+
+    dateItemFull: {
+      borderLeft: '1px solid white',
+      height: '36px',
+      borderTop: '1px solid white',
+      color: 'white',
+      position: 'relative',
+      top: '-5px',
+      cursor: 'pointer',
+      minWidth: '10px',
+      '&:hover': {
+        borderLeft: '1px solid #5ccfff',
+      },
+    },
+
+    dateItem: {
+      borderTop: '1px solid white',
+      color: 'white',
+      position: 'relative',
+      top: '-5px',
+      cursor: 'pointer',
+      minWidth: '10px',
+      '&:hover': {
```

Comment:
this is a little repetitive from the full version. Making a constant which shares commonality would be nice.